src/realm-settings/ProfilesTab.tsx

```diff
@@ -65,20 +80,18 @@ export const ProfilesTab = () => {
     },
   });
 
-  function cellFormatter(name?: IFormatterValueType) {
-    return (
-      <>
-        <Link to={""} key={`link-${name}`}>
-          {name}
-        </Link>{" "}
-        {profiles?.globalProfiles?.length! > 0 && (
-          <Label key={`label-${name}`} color="blue">
-            {t("global")}
-          </Label>
-        )}
-      </>
-    );
-  }
+  const cellFormatter = (row: ClientProfile) => (
+    <Link to={""} key={`label-${row.name}`}>
```

Comment:
When providing a `key` prop it only has to be unique within the context it is used. I think you can pass `row.name` directly to achieve the same result.
```suggestion
    <Link to={""} key={row.name}>
```